indico/modules/events/layout/controllers.py

```diff
@@ -54,21 +55,58 @@ def _render_menu_entries(event, show_hidden=False):
     return tpl.menu_entries(menu_entries_for_event(event, show_hidden=show_hidden))
 
 
+def _logo_data(event):
+    return {
+        'url': url_for('event_images.logo_display', event.as_legacy),
+        'file_name': event.logo_metadata['file_name'],
```

Comment:
am i the only one who thinks `file_name` (900k google results) is much uglier than `filename` (170M google results)?